AISKU/src/AISku.ts

```diff
@@ -259,8 +261,40 @@ export class AppInsightsSku implements IApplicationInsights {
                     _self.emptyQueue();
                     _self.pollInternalLogs();
                     _self.addHousekeepingBeforeUnload(_self);
+
+                    _addUnloadHook(onConfigChange(cfgHandler, () => {
+                        if (!_throttleMgr){
+                            _throttleMgr = new ThrottleMgr(_core);
```

Comment:
As with the config sync, maybe just move this to line 264 (before the onConfigChange) so that we don't need the if check.